buildconfig/CMake/WindowsNSIS.cmake

```diff
@@ -144,50 +156,33 @@ install ( FILES ${QT_PLUGIN_DIR}/sqldrivers/qsqlite4.dll DESTINATION plugins/qt4
 ###########################################################################
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/buildconfig/CMake/Packaging/launch_mantidplot.bat DESTINATION bin )
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/buildconfig/CMake/Packaging/launch_mantidplot.vbs DESTINATION bin )
-install ( FILES ${PROJECT_BINARY_DIR}/mantidpython.bat.install DESTINATION bin 
-    RENAME mantidpython.bat )
-
-###########################################################################
-# Icons for shortcuts
-###########################################################################
-install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/images/mantid_notebook.ico DESTINATION bin )
```

Comment:
As noted in previous comment. I think we'll need to keep the notebook stuff for reflectometry group.